### ISSUE TITLE
social icons do not display properly in user when browser small

### ISSUE THREAD

**Issue by yangeorget** [pledgcorporate/web#8]
maybe because we don't have 3 real columns

**Comment by xavigonz** [comment]
I am not able to see the problem

**Comment by yangeorget** [comment]
![image](https://cloud.githubusercontent.com/assets/1328291/13010306/80e22a4a-d1a1-11e5-985e-fff73c74b512.png)

**Comment by xavigonz** [comment]
You mean the bottom margin, right?

**Comment by yangeorget** [comment]
No. I  mean that there is enough space to display them on the right of the details instead of below.
